sql/mysql_db_defaultdata.sql

```diff
@@ -54,6 +54,7 @@ INSERT INTO `configuration` (`name`, `value`, `type`, `public`, `category`, `des
 ('team_column_width', '0', 'int', '0', 'Display', 'Maximum width of team column on scoreboard. Leave 0 for no maximum.'),
 ('enable_printing', '0', 'bool', '1', 'Misc', 'Enable teams and jury to send source code to a printer via the DOMjudge web interface.'),
 ('registration_category_name', '""', 'string', '1', 'Misc', 'Team category for users that register themselves with the system. Self-registration is disabled if this field is left empty.'),
+('registration_allow_affiliation', '0', 'bool', '1', 'Misc', 'Whether to allow to select / add an affiliation during registration.'),
```

Comment:
Do we want to have a separate config flag for this or rather use the existing show_affiliations? I am a bit torn, but we have a recent increase in flags...

Reply:
Hmmmm I’m also torn. I can’t think of a reason where you would show affiliations but not ask them during registration so I am leaning towards using the existing flag.